### ISSUE TITLE
the performance goes down after converting the yolov3 model into IR models

### ISSUE THREAD

**Issue by lihengMo** [openvinotoolkit/openvino#332]
Hi

I followed the official guideline and successfully converted the yolov3 model that trained by my own dataset into IR model and worked on the NCS. But I found that the performance become worse compared with original tensorflow model. with the original tensorflow model, I can get a mAP@0.5 around 95% on my test set but wile using the IR model detected on the NCS is just 63%. the problem is that the prediction of the boxes' coordinates is not good. the overlap is low. Is there any idea what leads to the performance goes down and how can I optimize the IR model to make the predicted boxes more accurently? 

thank you.

**Comment by ilyachur** [comment]
Hi @lihengMo,

What plugin do you use for network inference?
Is the issue still relevant for the last OpenVINO version?

**Comment by AnastasiaKazantaeva** [comment]
It seems that the issues is not actual anymore as no response. Closing it. Feel free to reopen it or create a new one.
